src/components/SideBar.js

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef } from 'react';
+import angleRight from '../assets/angle-right.svg';
+import closeMark from '../assets/close-mark.svg';
+import SideNav from './SideNav.js';
+import withClickOutside from './WithClickOutside.js';
+
+const SideBar = forwardRef(({ open, setOpen }, ref) => {
+  return (
+    <div className="relative z-10">
+      <SideNav styles="hidden md:flex lg:flex xl:flex" />
+      <div ref={ref} className="flex">
+        {open && <SideNav styles="fixed" />}
+        <button
+          type="button"
```

Comment:
The icon to open the sidebar would be better if we used the same hamburger as the one used on the navigation bar. Due to the fact that this sidebar replace the navigation bar in functionality